agent/acs/handler/heartbeat_handler.go

```diff
@@ -80,16 +82,17 @@ func (heartbeatHandler *heartbeatHandler) handleHeartbeatMessage() {
 
 func (heartbeatHandler *heartbeatHandler) handleSingleHeartbeatMessage(message *ecsacs.HeartbeatMessage) error {
 	// TestHandlerDoesntLeakGoroutines unit test is failing because of this section
-	// This should all move under a separate healthcheck handler
 
 	// Agent will run healthchecks triggered by ACS heartbeat
-	// healthcheck results will be sent on to TACS
-	//dockerHealthcheck := doctor.NewDockerRuntimeHealthcheck(heartbeatHandler.taskEngine.TaskEngineClient())
-	//heartbeatHandler.doctor.AddHealthcheck(dockerHealthcheck)
-	//go func() {
-	//	checkResult := heartbeatHandler.doctor.RunHealthchecks()
-	//	seelog.Debugf("handler healthcheck result: %v", checkResult)
-	//}()
+	// healthcheck results will be sent on to TACS, but for now just to debug logs.
+	go func() {
+		heartbeatHandler.doctor.RunHealthchecks()
+		requestMessage, err := heartbeatHandler.doctor.GetPublishInstanceStatusRequest()
+		if err != nil {
+			seelog.Errorf("Unable to get PublishInstanceStatusRequest message: %v", err)
```

Comment:
nit: should we return here? can be done in a different PR

Reply:
I'll add this in the integ test pr that's incoming.